exercises/concept/ellens-alien-game/.docs/introduction.md

```diff
@@ -0,0 +1,104 @@
+# Introduction
+
+## Classes
+
+It is time to get to one of the core paradigms of C++: object-oriented programming (OOP).
+OOP is centered around `classes` - user-defined types of data with their own set of related functions.
+We will start with the basics and will cover more advanced topics further down the syllabus tree.
+
+### Members
+
+Classes can have **member variables** and **member functions**.
```

Comment:
this might be a good point to mention that in this syllabus "member functions" and "methods" are synonyms. Otherwise we have two terms that might confuse the students.

Reply:
I brought this up as well.  I think we either define `method` as a name that is used for functions bound to a class, or we omit mention of methods to avoid confusion.  We could call it out in a note maybe?

Reply:
I hope I can purge all mentions of methods to avoid this issue. Did I miss any ...?
Quick search for the repository only offers 93 files... Hm. Not happy about that.

Reply:
I really have to go through the track to do this.